src/Symfony/Component/Serializer/Mapping/Loader/AttributeLoader.php

```diff
@@ -204,6 +204,16 @@ private function loadAttributes(\ReflectionMethod|\ReflectionClass|\ReflectionPr
         }
     }
 
+    private function setAttributeSerializedNameForGroups(SerializedName $annotation, AttributeMetadataInterface $attributeMetadata): void
```

Comment:
```suggestion
    private function setAttributeSerializedNameForGroups(SerializedName $attribute, AttributeMetadataInterface $attributeMetadata): void
```
(same for `setAttributeSerializedPathForGroups`)

Reply:
`setAttributeContextsForGroups`'s corresponding argument still have `$annotation` as its name. Do I really need to rename it now?